src/rendering/texturecomponent.cpp

```diff
@@ -0,0 +1,111 @@
+/*****************************************************************************************
```

Comment:
excited for these nice helper functions! Do we consider this like and OpenSpaceTextureComponent with stuff specific to OpenSpace? or should this just maybe be a class in ghoul?

Reply:
Good question. I guess it might actually make more sense for it to live in Ghoul, since it's sort of a simplified interface to the `ghoul::opengl::Texture` class. Maybe @alexanderbock has an opinion?

Reply:
I think having it in OpenSpace makes more sense since we than have some additional options.  For example, this file uses the  ghoul::filesystem::File  watch mechanism, which requires a function call to happen every frame to trigger these changes.  If this class would be in Ghoul and a person using Ghoul and this class doesn't make the call, it would work unexpected.